get-involved.md

```diff
@@ -6,18 +6,15 @@ order: 2
 
 Interested? You can start working with us in a few quick steps:
 
-**1.** Come say hi on our {% include icon-matrix.html %} chat
-
+1. Come say hi on our {% include icon-matrix.html %} chat
```

Comment:
While this is obviously the actual solution, I just need to let you know that I couldn't get it to work when I was writing the copy. I was too sad to admit defeat so I _stylized_ my failure